packages/ui/src/helpers/auth.ts

```diff
@@ -233,3 +207,13 @@ export const getServiceFacade = ({ send, state }) => {
     validationErrors,
   };
 };
+
+export const getServiceFacade = ({ send, state }) => {
+  const sendEventAliases = getSendEventAliases(send);
+  const serviceContext = getServiceContextFacade(state);
+
+  return {
+    ...sendEventAliases,
+    ...serviceContext,
+  };
+};
```

Comment:
I split off `getServiceFacade` to `sendEventAliases` and `getServiceContextFacade` for a tiny optimization in angular:

https://github.com/aws-amplify/amplify-ui/blob/74f211a78c1c25587253cf5953c60164fc0dd504/packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts#L44-L49

This avoids recomputation of `sendEventAliases` which is constant once service starts.

Reply:
Good call!  Totally agree.